.github/workflows/deploy-lego-bricks-storybook.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and Publish lego-bricks Storybook to GitHub Pages
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '22.x'
```

Comment:
Yeah we should bump the version for the webapp. But this file is actually unused, as I ended up hosting the Storybook using Vercel instead of github pages.
I will delete it now:)